index.html

```diff
@@ -5,9 +5,37 @@
         <link rel="icon" type="image/svg+xml" href="/jenkins.svg" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Jenkins infra-statistics</title>
+        <!-- Load necessary scripts for jio-navbar -->
+        <script type="module" src="https://unpkg.com/@jenkinsci/jenkins-io-components?module"></script>
+        <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.6.0/webcomponents-loader.js"></script>
+        <script src="https://unpkg.com/lit@2.4.0/polyfill-support.js"></script>
+        <script
+            data="ionicons"
+            src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.5.2/ionicons/ionicons.esm.js"
+            type="module"
+        ></script>
+        <script
+            data="ionicons"
+            nomodule=""
+            src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.5.2/ionicons/ionicons.js"
+        ></script>
     </head>
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
     </body>
 </html>
+
+<!-- <!doctype html>
```

Comment:
remove commented out code (you have git history)

applies throughout

Reply:
Addressed in https://github.com/jenkins-infra/stats.jenkins.io/pull/18